ops/aws/canary/lambda/pkg/test/scenarios_test.go

```diff
@@ -4,18 +4,35 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"strings"
 	"testing"
 
 	"github.com/filecoin-project/bacalhau/ops/aws/canary/pkg/models"
 	"github.com/filecoin-project/bacalhau/ops/aws/canary/pkg/router"
+	"github.com/filecoin-project/bacalhau/pkg/ipfs"
 	"github.com/filecoin-project/bacalhau/pkg/node"
 	"github.com/filecoin-project/bacalhau/pkg/requesternode"
 	testutils "github.com/filecoin-project/bacalhau/pkg/test/utils"
 	"github.com/stretchr/testify/require"
 )
 
 func TestScenarios(t *testing.T) {
-	stack, _ := testutils.SetupTest(context.Background(), t, 1, 0, false, node.NewComputeConfigWithDefaults(), requesternode.NewDefaultRequesterNodeConfig())
+	stack, _ := testutils.SetupTest(context.Background(), t, 3, 0, false, node.NewComputeConfigWithDefaults(), requesternode.NewDefaultRequesterNodeConfig())
+
+	os.Setenv("BACALHAU_ENVIRONMENT", "test")
+	t.Logf("BACALHAU_ENVIRONMENT: %s", os.Getenv("BACALHAU_ENVIRONMENT"))
+
+	swarmAddresses, err := stack.Nodes[0].IPFSClient.SwarmAddresses(context.Background())
+	require.NoError(t, err)
+	// Need to set the swarm addresses for getIPFSDownloadSettings() to work in test
+	os.Setenv("BACALHAU_IPFS_SWARM_ADDRESSES", strings.Join(swarmAddresses, ","))
+	t.Logf("BACALHAU_IPFS_SWARM_ADDRESSES: %s", os.Getenv("BACALHAU_IPFS_SWARM_ADDRESSES"))
+
+	// Add file to devstack IPFS
+	cid, err := ipfs.AddFileToNodes(context.Background(), "data.tar.gz", stack.IPFSClients()...)
```

Comment:
We need to add this test file manually because devstack, for some reason, wouldn't pull it from web3.storage